Dart/src/com/jetbrains/lang/dart/folding/DartFoldingBuilder.java

```diff
@@ -258,4 +261,32 @@ private static void foldTypeArguments(@NotNull final List<FoldingDescriptor> des
       }
     }
   }
+
+  private static void foldMultilineStrings(@NotNull final List<FoldingDescriptor> descriptors,
+                                           @NotNull final PsiElement root) {
+    for (PsiElement dartString : PsiTreeUtil.findChildrenOfType(root, DartStringLiteralExpression.class)) {
```

Comment:
I see what you mean, now that you point it out. I'm still adjusting to the way IDEA avoids using the visitor pattern. Can't quite get used to all the different alternatives.